### PR TITLE
feat: update dependencies, including react-styleguidist@10.6.2

### PR DESCRIPTION
**Pull request by stonebk**

### CODE REVIEW

**Comment by stonebk** [comment]
Published and tested a prerelease: `create-react-styleguide@prerelease`

File: package.json [hunk 1]
```diff
@@ -38,37 +38,37 @@
...
+    "jest": "^25.1.0",
+    "ora": "^4.0.3",
+    "prettier": "^1.19.1",
+    "react-styleguidist": "^10.6.2",
+    "resolve": "^1.15.1",
     "resolve-pkg": "^2.0.0",
-    "rimraf": "^2.7.1",
+    "rimraf": "^3.0.2",
     "run-parallel": "1.1.9",
     "run-series": "1.1.8",
```
Comment (evocateur) [inline package.json]: Any particular reason `run-series` and `run-parallel` have exact versions instead of caret ranges?

File: package.json [hunk 2]
```diff
@@ -38,37 +38,37 @@
...
-    "prettier": "^1.18.2",
-    "react-styleguidist": "^9.1.16",
-    "resolve": "^1.12.0",
+    "cross-spawn": "^7.0.1",
+    "eslint": "^6.8.0",
+    "inquirer": "^7.0.4",
+    "jest": "^25.1.0",
+    "ora": "^4.0.3",
+    "prettier": "^1.19.1",
+    "react-styleguidist": "^10.6.2",
```
Comment (evocateur) [inline package.json]: _No_ other changes necessary for a major release? Did they just bump their minimum node version or something?

File: package.json (hunk 1 above)
Comment (stonebk) [reply package.json]: Huh, not sure why that is. I'll go ahead and update that.

File: package.json (hunk 2 above)
Comment (stonebk) [reply package.json]: Yeah, they bumped a couple minimum versions:

https://github.com/styleguidist/react-styleguidist/releases/tag/v10.0.0

Breaking changes:
> Node 8.9 is the lowest supported version
> React 16.8 is the lowest supported version

File: package.json (hunk 1 above)
Comment (stonebk) [reply package.json]: Ok, updated -- didn't result in any outdated deps.

**Pull request merged**